avm/res/app/managed-environment/main.bicep

```
@@ -334,6 +349,23 @@ type certificateKeyVaultPropertiesType = {
   keyVaultUrl: string
 }?
 
+type certificateObjectType = {
+  @description('Optional. The name of the certificate.')
+  name: string?
+
+  @description('Optional. The type of the certificate.')
+  certificateType: ('ServerSSLCertificate' | 'ImagePullTrustedCA')?
+
+  @description('Optional. The value of the certificate. PFX or PEM blob.')
+  certificateValue: string?
+
+  @description('Optional. The password of the certificate.')
+  certificatePassword: string?
+
+  @description('Optional. A key vault reference.')
+  certificateKeyVaultProperties: certificateKeyVaultPropertiesType
+}?
```

```suggestion
}
```
`?` is ideally specified at parameter/property, not the type